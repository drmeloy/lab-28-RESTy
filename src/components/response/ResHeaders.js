import React from 'react';
import PropTypes from 'prop-types';

const ResHeaders = ({ username, password, token }) => {
  return (
    <details>
      <summary>"Headers" : </summary>
      <p>Username: {username}</p>
      <p>Password: {password}</p>
      <p>Bearer Token: {token}</p>
    </details>
  );
};

ResHeaders.propTypes = {

};

export default ResHeaders;
