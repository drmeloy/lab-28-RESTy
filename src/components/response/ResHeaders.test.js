import React from 'react';
import { shallow } from 'enzyme';
import ResHeaders from './ResHeaders';

describe('ResHeaders component', () => {
  it('renders ResHeaders', () => {
    const wrapper = shallow(<ResHeaders />);
    expect(wrapper).toMatchSnapshot();
  });
});
