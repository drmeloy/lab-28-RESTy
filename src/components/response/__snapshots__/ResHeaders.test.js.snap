// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResHeaders component renders ResHeaders 1`] = `
<details>
  <summary>
    "Headers" : 
  </summary>
  <p />
</details>
`;
